cou/steps/plan.py

```diff
@@ -237,6 +243,37 @@ def _verify_highest_release_achieved(analysis_result: Analysis) -> None:
         )
 
 
+def _verify_nova_cloud_controller_scheduler_default_filters(
+    args: CLIargs, analysis_result: Analysis
+) -> None:
+    """Verify scheduler_default_filters option is set correctly for each OpenStack release.
+
+    :param args: CLI arguments
+    :type args: CLIargs
+    :param analysis_result: Analysis result.
+    :type analysis_result: Analysis
+    """
+    if args.upgrade_group != CONTROL_PLANE:
```

Comment:
Upgrading the whole cloud (not passing an upgrade group) should also have this verify IMO.
A comment here is also good for future travelers explaining that data_plane already upgraded nova cntroller
```suggestion
    if args.upgrade_group not in {CONTROL_PLANE, None}:
```

Reply:
Isn't your suggested code change contradictory to what you wrote? :thinking:

Reply:
Thanks! that's been fixed. (it's very easy to forget that `None` means upgrade the whole cloud)